speechmatics_flow/client.py

```diff
@@ -450,14 +469,23 @@ async def run(
             consumer/producer tasks.
         """
         self.seq_no = 0
-        self._language_pack_info = None
         self.conversation_config = conversation_config
         self.audio_settings = audio_settings
 
         await self._init_synchronization_primitives()
 
         extra_headers = {}
-        auth_token = await get_temp_token(self.connection_settings.auth_token)
+        auth_token = self.connection_settings.auth_token
+        # Do not generate a JWT when connecting to a local server
+        local_servers = ("localhost", "127.0.0.1", "0.0.0.0")
+        if (
+            not any(
+                local_server in self.connection_settings.url
```

Comment:
Strictly, all of `127.0.0.0/8` is localhost, maybe there's a library function for this?

Reply:
Good point. Reverted this change and went with the option for setting --generate-temp-token false from CLI